spec/lib/checking_account_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper.rb'
+
+describe BankAccount::CheckingAccount do 
+  
+  context "CheckingAccount" do
+
+    it "knows about init from parent class" do
+      expect(BankAccount::CheckingAccount).to respond_to :new
+    end
+  end
+
+  context "withdraw methods methods" do
+    let(:checking_account) {BankAccount::CheckingAccount.new(6, 600)}
+
+    it "withdraws correct amount + $1" do
```

Comment:
Would use "fee" in your `it` descriptor rather than `$1` so that if the fee amount changed you wouldn't have to change your test